ApiExtractor/abstractmetabuilder.cpp

```diff
@@ -34,20 +34,20 @@
 #include "parser/parser.h"
 #include "parser/tokens.h"
 
-#include <QDebug>
-#include <QFile>
-#include <QFileInfo>
-#include <QTextCodec>
-#include <QTextStream>
-#include <QVariant>
-#include <QTime>
-#include <QQueue>
-#include <QDir>
+#include <QtCore/QDebug>
```

Comment:
Dual Qt4/Qt5 support isn't a necessity. It can be done if it isn't too much work, but seeing as upstream doesn't care much about this, I expect we may be left with a fork here anyways :( .